docs/sources/tempo/configuration/_index.md

```diff
@@ -1460,6 +1556,48 @@ parquet_dedicated_columns: <list of columns>
       [scope: <string>]
 ```
 
+### Compaction config
+
+The compaction configuration block is used by the compactor, backend scheduler, and backend worker.
+Refer to the [Compactor](#compactor) section for the full list of compaction configuration options.
+
+```yaml
+# Optional. Duration to keep blocks. Default is 14 days (336h).
```

Comment:
is this value in hours or days? The comment (336h) makes me think that this value is in hours. 

In the previous section, the defaults are listed as inline with the options: `[enable_inet6: <bool> | default = false ]`. When we're showing the default value in this section, the defaults are shown in the text. Is this a difference in how the configuration options are or just a variation in how we're showing defaults? Should (can?) we be consistent?